Libraries/PeriphDrivers/Source/WDT/wdt_me18.c

```diff
@@ -153,3 +153,36 @@ void MXC_WDT_ClearIntFlag(mxc_wdt_regs_t *wdt)
 {
     MXC_WDT_RevB_ClearIntFlag((mxc_wdt_revb_regs_t *)wdt);
 }
+
+int MXC_WDT_SetClockSource(mxc_wdt_regs_t *wdt, mxc_wdt_clock_t clock_source)
+{
+    const uint8_t clock_source_num = 8;
+    uint8_t idx = 0;
+    uint8_t instance = 0;
+    mxc_wdt_clock_t clock_sources[2][8] = {
```

Comment:
Initialize with `clock_source_num` instead of `8`

```C
mxc_wdt_clock_t clock_sources[2][clock_source_num] = ...
```

Reply:
Hmm strange...  should work as long as it's `const`.  What's the error?

Reply:
It is "error: variable-sized object may not be initialized".

![image](https://github.com/Analog-Devices-MSDK/msdk/assets/94184469/c2b3ec24-5db8-40f5-8ee9-397b075033d0)